SeleniumDemoQA/SeleniumDemoQA/Pages/FormPage.cs

```diff
@@ -0,0 +1,124 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SeleniumDemoQA.Pages
+{
+    internal class FormPage
+    {
+        public IWebDriver _driver;
+        public IJavaScriptExecutor _js;
+        By confirmationModalElement = By.Id("example-modal-sizes-title-lg");
+
+        public FormPage(IWebDriver driver)
+        {
+            _driver = driver;
+            _js = (IJavaScriptExecutor)_driver;
+        }
+
+        public void ScrollTo(IWebElement element)
+        {
+            _js.ExecuteScript("arguments[0].scrollIntoView(true);", element);
+        }
+
+        public void FillInput(By selector, string value)
+        {
+            var webElement = _driver.FindElement(selector);
+            ScrollTo(webElement);
+            webElement.SendKeys(value);
+        }
+        public void ClickElement(By selector)
+        {
+            var webElement = _driver.FindElement(selector);
+            ScrollTo(webElement);
+            webElement.Click();
+        }
+        public void FillInputAndEnter(By selector, string value)
+        {
+            var webElement = _driver.FindElement(selector);
+            ScrollTo(webElement);
+            webElement.SendKeys(value);
+            webElement.SendKeys(Keys.Enter);
+        }
+        public void ClickWithoutScroll(By selector)
+        {
+            var webElement = _driver.FindElement(selector);
+            webElement.Click();
+        }
+        public void SelectText(By selector, string text)
```

Comment:
Name can be SelectByText but not SelectText as this method selects not a text, but one of "Select" element option